<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the public 'sylius.behat.context.ui.shop.checkout.thank_you' shared service.

include_once \dirname(__DIR__, 4).'/vendor/behat/behat/src/Behat/Behat/Context/Context.php';
include_once \dirname(__DIR__, 4).'/vendor/sylius/sylius/src/Sylius/Behat/Context/Ui/Shop/Checkout/CheckoutThankYouContext.php';

return $this->services['sylius.behat.context.ui.shop.checkout.thank_you'] = new \Sylius\Behat\Context\Ui\Shop\Checkout\CheckoutThankYouContext(($this->privates['sylius.behat.page.shop.order.thank_you'] ?? $this->load('getSylius_Behat_Page_Shop_Order_ThankYouService.php')), ($this->privates['sylius.behat.page.shop.account.order.show'] ?? $this->load('getSylius_Behat_Page_Shop_Account_Order_ShowService.php')), ($this->services['sylius.repository.order'] ?? $this->getSylius_Repository_OrderService()));
